### PR TITLE
Add text2vec-jinaai module

### PR DESCRIPTION
**Pull request by JoanFM**
### What's being changed:

Hey Weaviate team!

We just launched [Jina Embeddings](https://jina.ai/embeddings/) and would love to add a module that would allow users to use them as a module in Weaviate.

Thanks!

### Review checklist

- [x] Documentation has been updated, if necessary. Link to changed documentation: https://github.com/weaviate/weaviate-io/pull/1266
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

**Comment by JoanFM** [comment]
> To avoid any confusion in the future about your contribution to Weaviate, we work with a [Contributor License Agreement](https://weaviate.io/service/contributor-license-agreement). If you agree, you can simply add a comment to this PR that you agree with the CLA so that we can merge.beep boop - the Weaviate bot 👋🤖PS:Are you already a member of the [Weaviate Slack channel?](https://weaviate.io/slack)

I agree

**Review by antas-marcin — DISMISSED** [review]

File: modules/text2vec-jinaai/clients/meta.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clients
+
+func (v *vectorizer) MetaInfo() (map[string]interface{}, error) {
+	return map[string]interface{}{
+		"name":              "JinaAI Module",
+		"documentationHref": "",
```
Comment (antas-marcin) [inline modules/text2vec-jinaai/clients/meta.go:17]: would be cool to return https://jina.ai/embeddings/ in `documentationHref`

Comment (JoanFM) [reply modules/text2vec-jinaai/clients/meta.go:17]: Makes sense, I will

Comment (JoanFM) [reply modules/text2vec-jinaai/clients/meta.go]: Done!

**Comment by antas-marcin** [comment]
Please also whitelist `X-JinaAI-Api-Key` header [here](https://github.com/weaviate/weaviate/blob/stable/v1.22/usecases/config/config_handler.go#L255)

**Comment by JoanFM** [comment]
> Please also whitelist `X-JinaAI-Api-Key` header [here](https://github.com/weaviate/weaviate/blob/stable/v1.22/usecases/config/config_handler.go#L255)

Done @antas-marcin , thanks for the guidance

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**